backend/api/models.py

```diff
@@ -1,3 +1,24 @@
+from django.conf import settings
 from django.db import models
 
 # Create your models here.
+from django.dispatch import receiver
+from django_rest_passwordreset.signals import reset_password_token_created
+from django.core.mail import send_mail
+
+from templates.consts import *  
+
+
+@receiver(reset_password_token_created)
+def password_reset_token_created(sender, instance, reset_password_token, *args, **kwargs):
```

Comment:
Why this code is in the `models` file? Is this model?

Reply:
It's not. The guy who wrote it had placed it in `models` =) works but is not appropriate. Apparently it's best practice to put _signal handlers_ in `api/signals/handlers.py`

Reply:
placing it somewhere other than models or views fails to receive the signal and the approach i mentioned above failed in the first try.